doc/source/releases/21.01_announce_user.rst

```diff
@@ -0,0 +1,135 @@
+
+===========================================================
+January 2021 Galaxy Release (v 21.01)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+* Galaxy Markdown enhancements including integration of visualizations - allow embedding
+  visualizations in Galaxy Markdown (workflow invocation reports and pages) with
+  Markdown editor support and allow exporting workflow reports directly to Galaxy Pages.
+  `Pull Request 10241`_, `Pull Request 10288`_, `Pull Request 10404`_
+
+* History panel rewrite (available in the UI as a beta option).
+  `Pull Request 8441`_, `Pull Request 11071`_, `Pull Request 10971`_
+
+* Allow organizing the panel via EDAM ontology annotations.
+  (thanks to `@hexylena <https://github.com/hexylena>`__).
+  `Pull Request 10592`_
+
+* Large overhaul over history import and export - including improved user experience, APIs, and allowing importing and exporting
+  to file sources plugins (including FTP).
+  `Pull Request 11054`_, `Pull Request 10993`_
+
+* Various substantial improvements to data library user interface and performance
+  (thanks to `@OlegZharkov <https://github.com/OlegZharkov>`__).
+  `Pull Request 10204`_, `Pull Request 11099`_,
+  `Pull Request 10557`_, `Pull Request 10558`_,
+  `Pull Request 10807`_
+
+* UI & API for Supporting Best Practice Workflows
+  `Pull Request 10988`_, `Pull Request 11025`_
+* Add author and license metadata and corresponding web microdata annotations for tools
+  and workflows.
+  `Pull Request 10408`_
+
```

Comment:
todo @hexylena add workflow invocation progress thing for @mvdbeek . thanks for the reminder of sth I missed!

Reply:
Did not intend to forget this -- sometimes just can't tell if something is interesting from the PR titles / multiple minor PRs to enhance something get lost sometimes :(

please let me know if I'm misisng other stuff!